great-idea/css/index.css

```diff
@@ -3,64 +3,471 @@
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+h2 {
+  font-family: "Anton", sans-serif;
+}
+
+.static-navbar {
+  /* position: fixed; */
+  top: 0;
+  width: 100%;
+  /* background-color: rgb(0, 93, 189); */
+  z-index: 5;
+}
+
+nav {
+  display: block;
+  width: 880px;
+  height: 100px;
+  margin: 0 auto;
+  position: relative;
+}
+
+nav a {
+  text-decoration: none;
+  color: black;
+  margin: 35px 31px 0 31px;
+  font-size: 14px;
+  display: inline-block;
+}
+
+nav a:hover {
+  color: rgb(176, 177, 178);
+}
+
+nav a:first-child {
+  margin-left: 0;
+}
+
+nav div {
+  display: inline-block;
+  position: absolute;
+  right: 0;
+}
+
+img {
+  margin-top: 25px;
+}
+
+.website-wrapper {
+  width: 880px;
+  margin: 130px auto 0 auto;
+  z-index: 1;
+}
+
+header {
+  border-bottom: 1px solid black;
+  position: relative;
+  height: 490px;
+}
+
+header div {
+  display: inline-block;
+  text-align: center;
+  margin: 70px 0 0 85px;
+}
+
+header div h1 {
+  font-size: 80px;
+  margin-bottom: 30px;
+}
+
+header div h1 span {
+  display: block;
 }
 
-h1, h2, h3, h4, h5 {
-    font-family: 'Bangers', cursive;
-    letter-spacing: 1px;
-    margin-bottom: 15px;
+header div a {
+  text-decoration: none;
+  color: black;
+  border: 1px solid black;
+  padding: 4px 48px;
+}
+
+header img {
+  display: inline-block;
+  position: absolute;
+  right: 0;
+}
+
+.section-one {
+  margin-top: 39px;
+}
+
+.section-one div {
+  width: 430px;
+  height: 100px;
+  display: inline-block;
+}
+
+.section-one div p {
+  line-height: 1.16;
+}
+
+.section-one div:first-child {
+  margin-right: 10px;
+}
+
+.middle-img {
+  margin-top: 38px;
+}
+
+.section-two {
+  border-bottom: 1px solid black;
+  margin-top: 36px;
+  padding-bottom: 47px;
+}
+
+.section-two div {
+  display: inline-block;
+  width: 280px;
+  margin-right: 10px;
+}
+
+.section-two div p {
+  line-height: 1.16;
+}
+
+address {
+  margin-top: 38px;
+}
+
+address div {
+  line-height: 1.16;
+}
+
+address div span {
+  display: inline-block;
+  margin: 17px 0 18px 0;
+}
+
+footer {
+  width: 100%;
+  height: 183px;
+  text-align: center;
+}
+
+footer p {
+  margin-top: 54px;
+  font-size: 14px;
+}
+
+/* My code changes starts here */
+
+.wrapper {
+	max-width: 100%;
+	width: 1000px;
+	margin: auto;
+}
+
+.main-content {
+
+  display: flex;
+  width: 90%;
+  margin: 25px auto;
+  flex-flow: wrap;
+  justify-content: space-around;
+
+}
+
+.services-item {
+	display: flex;
+	align-items: flex-start;
+	flex-wrap: wrap;
+	margin: auto;
+	 /* margin: 50px 50px 1px 10px;  */
+	padding: 0px 0px 30px 0px; 
+	border-bottom: 1px solid gray;
+	width: 90%;
+	
+}
+
+.services-item .serv-child h2 {
+	
+	font-size: 20px;
+	font-family: "Bangers", cursive;
+	letter-spacing: 1px;
+	margin-bottom: 15px;
+	width: 80px;
+}
+
+
+
+.bottom-page-sec {
+  display: flex;
+  width: 90%;
+  justify-content: space-around;
+  margin: 30px 0px 70px 70px;
+}
+/* Middle iteams */
+
+.digital-design-item {
```

Comment:
in the future look into using dual class identifiers so that you can format many boxes with one class